### PR TITLE
Recipe details

### LINKED ISSUE #6: Public recipe list [3pts]
# Public recipe list [3pts]

- Should display a list of all public recipes ordered by newest as in the wireframe.
- Should lead to recipe details.

image

### LINKED ISSUE #7: Recipe details [3pts]
# Recipe details [3pts]

- If the recipe is public or the user is the owner of the recipe, should display the recipe details as in the wireframe.
- If the user is the owner of the recipe, should lead to the form that allows user to add new food.

image

### PR DESCRIPTION
**Pull request by eapenzacharias**
## Recipe details
- If the recipe is public or the user is the owner of the recipe, should display the recipe details as in the wireframe.
- If the user is the owner of the recipe, should lead to the form that allows the user to add new food.

## Public recipe list
- Should display a list of all public recipes ordered by newest as in the wireframe.
- Should lead to recipe details.
- Added link to navbar

### CODE REVIEW

File: app/controllers/recipes_controller.rb [hunk 1]
```diff
@@ -13,14 +15,41 @@ def index
...
+      else
+        flash[:fail] = 'recipe deletion unsucessful.'
+      end
+    end
+    redirect_to recipes_path
+  end
+
+  def fetch_recipe
+    @recipe = Recipe.find_by_id(params[:recipe_id])
+  end
```
Comment (piero-vic) [inline app/controllers/recipes_controller.rb]: [OPTIONAL] I think this function could be a private function. Please correct me if I'm wrong.

File: app/views/recipe_foods/new.html.erb [hunk 2]
```diff
@@ -0,0 +1,30 @@
+<div class="w-full">
+  <h1 class="font-bold text-4xl">New Inventory Item</h1>
```
Comment (piero-vic) [inline app/views/recipe_foods/new.html.erb]: You could change this title to "New Ingredient".

**Review by piero-vic — APPROVED** [review]
Hi @eapenzacharias 

## Status: Appoved :fire:

All the functionality is well implemented but there are a couple of issues we can address. Make sure to check them before merging the branch.

**Review by lu-jim — APPROVED** [review]
Hi @eapenzacharias ,

Your feature is complete! There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉

Cheers and Happy coding!👏👏👏

**Pull request merged**